### PR TITLE
Dan order fixes

### PR DESCRIPTION
**Pull request by dHofstede**
Fixed a number of issues:
-used literal index values for address when making orders in mock data to correct an issue where some addresses were ending up as null values
-fixed an order which was processed but not shipped (an impossible state given the assumptions that we've made)
-corrected game names fon mock orders from old game names ("test" and "rwar") to proper mock data names.
-provided a link to order hostry in the layout
-added cvgs logo in shipping label

### CODE REVIEW

File: SeaSharpe_CVGS/SeaSharpe_CVGS/Migrations/Configuration.cs [hunk 1]
```diff
@@ -258,8 +258,8 @@ private Order MakeOrder(string member, string aprover, string orderPlacementDate
             {
                 Member = db.Members.First(m => m.User.UserName == member),
                 Aprover = db.Employees.FirstOrDefault(e => e.User.UserName == aprover),
-                BillingAddress = db.Addresses.OrderBy(c => c.City).Skip(billingAddressIndex).FirstOrDefault(),
-                ShippingAddress = db.Addresses.OrderBy(c => c.City).Skip(shippingAddressIndex).FirstOrDefault(),
+                BillingAddress = db.Addresses.Find(billingAddressIndex),
```
Comment (BlackthornYugen) [inline SeaSharpe_CVGS/SeaSharpe_CVGS/Migrations/Configuration.cs]: Billing address index is not an address id, and the find method is used to find an ID. You should be using skip and firstordefault on a query ordered by the ID. That way if the index is 0 you get the first address and if the index is 1 you get the second.

File: SeaSharpe_CVGS/SeaSharpe_CVGS/Views/Shared/_Layout.cshtml [hunk 2]
```diff
@@ -39,9 +39,10 @@
                 <ul class="nav navbar-nav">
                     <li>@Html.ActionLink("Friends/Family", "Index", "Friendship")</li>
                     <li>@Html.ActionLink("Wish List", "DisplayCurrentMemberWishlist", "Friendship")</li>
-                    <li>@Html.ActionLink("Library", "Index", "Home")</li> <!--library page not created yet-->
                     <li>@Html.ActionLink("Profile", "Edit", "User")</li>
+                    <li>@Html.ActionLink("Order History", "OrderHistory", "Order")</li> <!--library page not created yet-->
                     <li>@Html.ActionLink("Cart", "Cart", "Order")</li>   
+                    <li>@Html.ActionLink("Order History", "OrderHistory", "Order")</li> <!--library page not created yet-->
```
Comment (BlackthornYugen) [inline SeaSharpe_CVGS/SeaSharpe_CVGS/Views/Shared/_Layout.cshtml]: Two links to order history?

Comment (dHofstede) [inline SeaSharpe_CVGS/SeaSharpe_CVGS/Views/Shared/_Layout.cshtml]: I'm not aware of a second link to order history?

Comment (BlackthornYugen) [inline SeaSharpe_CVGS/SeaSharpe_CVGS/Views/Shared/_Layout.cshtml]: image

**Comment by BlackthornYugen** [comment]
+1 :+1:

**Comment by ndahlquist-cc** [comment]
+1

**Pull request merged**